demisto_sdk/commands/content_graph/parsers/repository.py

```diff
@@ -56,7 +65,7 @@ def iter_packs(self, packs_to_parse: Optional[List[str]] = None) -> Iterator[Pat
             Iterator[Path]: A pack path.
         """
         packs_folder: Path = self.path / PACKS_FOLDER
-        if packs_to_parse:
+        if packs_to_parse is not None:
```

Comment:
can you explain this change?

Reply:
The idea was to run `finalize` even if there are no packs to parse. I will revert this